drivers/dma/CMakeLists.txt

```diff
@@ -12,3 +12,5 @@ zephyr_library_sources_ifdef(CONFIG_DMA_NIOS2_MSGDMA	dma_nios2_msgdma.c)
 zephyr_library_sources_ifdef(CONFIG_DMA_SAM0		dma_sam0.c)
 zephyr_library_sources_ifdef(CONFIG_USERSPACE		dma_handlers.c)
 zephyr_library_sources_ifdef(CONFIG_DMA_MCUX_EDMA	dma_mcux_edma.c)
+zephyr_library_sources_ifdef(CONFIG_DMA_IPROC_PAX	dma_iproc_pax.c)
+zephyr_library_sources_ifdef(CONFIG_DMA_IPROC_PAX_V2	dma_iproc_pax.c)
```

Comment:
Hi, I think you should reshuffle the config options such that you don't have a repeated linke with the C file twice here (see below).

Reply:
Hi, 
There's a correction here, it should be

+zephyr_library_sources_ifdef(CONFIG_DMA_IPROC_PAX	dma_iproc_pax.c)
+zephyr_library_sources_ifdef(CONFIG_DMA_IPROC_PAX_V2	dma_iproc_pax_v2.c)

V1 and V2 are hardware needs to be programmed differently so the driver's are different.

So I feel keeping different CONFIG option is necessary here.

Reply:
Please look at updated patch.